include/avif/avif.h

```diff
@@ -1138,6 +1152,9 @@ typedef struct avifScalingMode
 //   call to avifEncoderAddImage().
 typedef struct avifEncoder
 {
+    // Defaults to AVIF_ENCODER_FULL_HEADER
+    avifEncoderHeaderStrategy headerStrategy;
```

Comment:
In anticipation of the libavif 1.0.0 release, the new member `headerStrategy` should be added to the end of the `avifEncoder` struct, after all the current members.

Reply:
I think it makes more sense to have it at the beginning of the struct than after the "Internals used by the encoder" section. If the purpose is to avoid breaking the API (ABI?), I thought the 1.0.0 release was a big change anyway, so it would be a good timing for struct field reordering.

Reply:
I am adding "Version 1.0.0 ends here" comments to the end of the `avifImage`, `avifDecoder`, and `avifEncoder` structs to help address this issue:
https://github.com/AOMediaCodec/libavif/pull/1510

Note: If the reduced HEIF header proposal has a good chance of being adopted, we can also add the new `avifEncoderHeaderStrategy headerStrategy` member to `avifEncoder` in the libavif 1.0.0 release. In version 1.0.0, it must be set to `AVIF_ENCODER_FULL_HEADER`.

Reply:
OK, then let's just wait after the 1.0.0 release. Thanks.